.github/workflows/image_build.yaml

```diff
@@ -16,5 +18,5 @@ jobs:
           TAG: "dev-latest"
```

Comment:
Maybe we don't care to publish anything besides amd64 for the `dev-latest` tag?

Reply:
i'm fine with all of them being pushed.